cmd/worker/main.go

```diff
@@ -78,6 +79,16 @@ func HandleKeyGeneration(ctx context.Context, t *asynq.Task) error {
 		"keyEDDSA": keyEDDSA,
 	}).Info("Key generation completed")
 
+	redis, err := storage.NewRedisStorage(config.AppConfig)
```

Comment:
I would suggest to have only one instance of redis and use it where needed

Reply:
This is the only one instance used in worker process which is an independent process with vultisigner one.

Reply:
Well, `HandleKeyGeneration` will be called for every single KeyGeneration task, that means we will be connect to redis and disconnect for every keygen task. 
Not best design 
so it will be better to define a `Service` struct, and keep `HandleKeyGeneration` as a method of `Service` struct,  and then hold a `redis` instance in `Service` , this way , the worker will only maintain a redis connection long term

Reply:
ok 👍